### PR TITLE
Include list of enabled add-ons in reports (closes #24).

### LINKED ISSUE #24: Reports should include list of installed add-ons.
https://developer.mozilla.org/en-US/Add-ons/Add-on_Manager/AddonManager#getAllAddons()

### PR DESCRIPTION
**Pull request by chuckharmston**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = {
   externals: [
     // Treat all sdk requires as importable commonjs modules.
     (context, request, callback) => {
-      if (/^sdk\//.test(request)) {
+      if (/^sdk\//.test(request) || /^resource\:\/\//.test(request)) {
```
Comment (pdehaan) [inline webpack.config.js]: Probably not as readable, but you could condense this into a single RegExp, something like:

```js
if (/^(sdk\/|resource:\/\/)/.test(request)) {
```

Although it's only a webpack build, so possibly not worth the complexity.

Comment (chuckharmston) [reply webpack.config.js]: Totally true, though I'd rather keep them explicitly separate for readability purposes.

**Comment by chuckharmston** [comment]
Updated to accommodate 7b5e503.

File: src/measurements/addons.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { AddonManager } from 'resource://gre/modules/AddonManager.jsm';
+import BaseMeasurement from '../lib/measurement';
+
+export default class Addons extends BaseMeasurement {
+  measure(resolve) {
+    AddonManager.getAllAddons(addons => {
+      resolve(
+        addons
+          .filter(addon => addon.isActive)
+          .map(addon => addon.id)
```
Comment (pdehaan) [inline src/measurements/addons.js:10]: Interesting fact: this includes all hidden built-in add-ons as well.

For example, I only have the Pulse add-on installed, but the ping includes ~13 add-ons (including Pocket, etc).

add-ons_manager_and_include_list_of_enabled_add-ons_in_reports__closes__24___by_chuckharmston_ _pull_request__67_ _mozilla_pulse_and_1____npm_bin__jpm_run_--addon-dir_build_-b_nightly__firefox-bin__and_index_js_ _build_webextension_survey_


browser_console_and_builds_user_sentiment_survey__closes__10__closes__18___by_chuckharmston_ _pull_request__82_ _mozilla_pulse

**Pull request merged**